src/modules/fetchcomment.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable import/prefer-default-export */
+const commentsload = document.querySelector('.commentsload');
```

Comment:
- [ ] Kindly enable eslint for it is not allowed to disable linters. linters help us write clean code therefore it is against microverse policy to disable linters.

Reply:
done